internal/app/service_test.go

```diff
@@ -313,32 +313,30 @@ func TestService_AddFunctionsPipelineForTopic(t *testing.T) {
 
 	transforms := []interfaces.AppFunction{tags.AddTags}
 
-	// This sets the Default Pipeline allowing to test for duplicate iD.
-	err := service.SetDefaultFunctionsPipeline(transforms...)
-	require.NoError(t, err)
+	defaultTopics := []string{"#"}
 
 	tests := []struct {
 		name        string
 		id          string
 		trigger     string
-		topic       string
+		topics      []string
 		transforms  []interfaces.AppFunction
 		expectError bool
 	}{
-		{"Happy Path", "123", TriggerTypeMessageBus, "#", transforms, false},
-		{"Happy Path Custom", "124", "CUSTOM TRIGGER", "#", transforms, false},
-		{"Empty Topic", "125", TriggerTypeMessageBus, " ", transforms, true},
-		{"No Transforms", "126", TriggerTypeMessageBus, "#", nil, true},
-		{"Default Id", interfaces.DefaultPipelineId, TriggerTypeMessageBus, "#", transforms, true},
-		{"Duplicate Id", "123", TriggerTypeMessageBus, "#", transforms, true},
-		{"HTTP Trigger Type", "127", TriggerTypeHTTP, "#", transforms, true},
+		{"Happy Path", "123", TriggerTypeMessageBus, defaultTopics, transforms, false},
+		{"Duplicate Id", "123", TriggerTypeMessageBus, defaultTopics, transforms, true},
```

Comment:
This test depends on pervious test to have run. Correct? This is not good practice as I can run this test by itself. This is way I used `service.SetDefaultFunctionsPipeline` above. Does loo like I had something else messaged up... ;-) `"Default Id", interfaces.DefaultPipelineId,` should have been the `"Duplicate Id"`